Dockerfile

```
@@ -0,0 +1,29 @@
+FROM node:14.17.4 AS restore
+WORKDIR /src
+COPY package.json yarn.lock ./
+RUN yarn
+COPY . .
+
+FROM restore AS verify-format
+RUN yarn verify-format
+
+FROM koalaman/shellcheck-alpine as verify-sh
+WORKDIR /src
+COPY ./*.sh ./
+RUN shellcheck -e SC1091,SC1090 ./*.sh
+
+FROM restore as build
+COPY . .
```

```suggestion
FROM restore as build
```

El `COPY . .` ya lo estamos haciendo en _restore_

---

Changed.